pkg/controller/storagecluster/update.go

```diff
@@ -82,7 +82,7 @@ func (c *Controller) rollingUpdate(cluster *corev1.StorageCluster, hash string)
 
 	// get the number of kvdb members which are unavailable in case of internal kvdb
 	numUnavailableKvdb, kvdbNodes, err := c.getKVDBNodeAvailability(cluster)
```

Comment:
do we even need to check KVDB node's availability in case of fresh install then?